app/components/CompanyValues.tsx

```diff
@@ -0,0 +1,145 @@
+// LIBRARY IMPORTS
+import {
+  Box,
+  Flex,
+  Heading,
+  HStack,
+  Icon,
+  Spacer,
+  Text,
+  VStack,
+} from "@chakra-ui/react";
+import Image from "next/image";
+import { FaRegClock, FaPeopleGroup, FaRegStar } from "react-icons/fa6";
+
+// LOCAL IMPORTS
+import gearBackground from "../../public/images/machine-gear-background.png";
+
+// VALUES COMPONENT DEFINITION
+export default function CompanyValues() {
+  return (
+    <>
+      <Image
+        src={gearBackground}
+        placeholder="blur"
+        fill
+        sizes="100vw"
+        style={{
+          objectFit: "cover",
+          zIndex: -1,
+        }}
+      />
+      <Box
+        bgImage={gearBackground}
+        backgroundRepeat="no-repeat"
+        backgroundSize="cover"
+        p={10}
+        bg="brand.backgroundDark"
+        opacity="90%"
+      >
+        <VStack spacing={8}>
+          <Heading color="white" textAlign="center" mb={4}>
+            {"Kenworthy's Values"}
+          </Heading>
+          <VStack
+            maxW="1400px"
+            bg="white"
+            borderRadius="2xl"
+            px={32}
+            py={24}
+            spacing={36}
+          >
+            <HStack spacing={12} align="center">
+              <VStack align="start" spacing={4}>
+                <Heading size="lg">Quality</Heading>
+                <Box
+                  bgGradient="linear(90deg, #DB0A40 0%, rgba(219, 10, 64, 0.04) 96%, rgba(219, 10, 64, 0) 100%)"
+                  w="full"
+                  h={1}
+                ></Box>
+                <Text fontWeight="bold" fontSize="lg">
+                  {
+                    "At Kenworthy Machine, producing quality parts to our customer's specifications is our most important job."
+                  }
+                </Text>
+                <Text fontSize="lg">
+                  {
+                    "Our team is dedicated to making sure your parts are correct an in tolerance every time. They use their expertise, as well as modern inspection equipment to achieve this goal. Upon request we can provide inspection reports for your parts with detailed measurements compared to drawing specifications. We also offer laser engraving to give your parts a professional appearance."
+                  }
+                </Text>
+              </VStack>
+              <Spacer />
```

Comment:
This `<Spacer />` component combined with the `spacing={12}` property set in the outermost `HStack` is what creates enough horizontal space between the text and the icon as per the wireframe. However, something about this is resizing the icon's container.

Reply:
In [ChakraUI docs](https://chakra-ui.com/docs/components/flex/usage#using-the-spacer) it says that _"Since Spacer renders a div, any gap value provided to the parent is applied to both sides of this component, and therefore make the gap appear doubled when the spacer is completely collapsed."_ I assume that it has impact on flex container with the icon because you give it width and height that gets ignored. 

What I've tried was to delete the spacer, set padding for flex container (so that it won't be ignored and give the nice padding as it appears on Figma) and replace width and height props for Icon with boxSize (I forgot to double-check if that's truly necessary). See the snippet bellow and the screenshot of what the icons and spacing looks like with these changes applied. You might still want to try to play with the horizontal spacing between the icon and the text.

```javascript
<HStack spacing={12} align="center">
  <VStack align="start" spacing={4}>
    <Heading size="lg">Quality</Heading>
      <Box
        bgGradient="linear(90deg, #DB0A40 0%, rgba(219, 10, 64, 0.04) 96%, rgba(219, 10, 64, 0) 100%)"
        w="full"
        h={1}
      >
      </Box>
      <Text fontWeight="bold" fontSize="lg">
        {
          "At Kenworthy Machine, producing quality parts to our customer's specifications is our most important job."
         }
      </Text>
      <Text fontSize="lg">
        {
          "Our team is dedicated to making sure your parts are correct an in tolerance every time. They use their expertise, as well as modern inspection equipment to achieve this goal. Upon request we can provide inspection reports for your parts with detailed measurements compared to drawing specifications. We also offer laser engraving to give your parts a professional appearance."
        }
      </Text>
    </VStack>
    {/* <Spacer /> */}
    <Flex
      // w={28}
      // h={28}
      p={4}
      bg="brand.backgroundDark"
      borderRadius="2xl"
      align="center"
      justifySelf="center"
    >
    <Icon
       as={FaRegStar}
      // w={24}
      // h={24}
      boxSize={24}
      color="white"
    />
  </Flex>
</HStack>

```  
![Screen Shot 2024-02-01 at 2 32 50 PM](https://github.com/wavecrestweb/kenworthy-machine/assets/109827834/9719fd7f-cf9e-4e3f-8851-bddb6046e479)